### ISSUE TITLE
Latex build fails with Sphinx 2

### ISSUE THREAD

**Issue by aronhelser** [blockdiag/sphinxcontrib-actdiag#3]
Sphinx 2 has moved to a new logging system, and it looks like actdiag hasn't been updated. 
Reported as a [sphinx issue](https://github.com/sphinx-doc/sphinx/issues/6673), it occurs in actdiag and blockdiag.

**Comment by aronhelser** [comment]
I can avoid the exception by patching sphinxcontrib/actdiag.py, line 253:
```
def on_builder_inited(self):
    if not hasattr(self.builder, 'warn'):
        self.builder.warn = print
```
so hopefully

**Comment by tk0miya** [comment]
Fixed by #4. Thank you for reporting.

**Comment by aronhelser** [comment]
Awesome! thank you so much!
